apps/me/.env.cloud-run.example

```
@@ -0,0 +1,10 @@
+# Cloud Run deployment secrets — populate and run scripts/cloud-run-env.sh
+# Copy to .env.cloud-run.prod or .env.cloud-run.staging and fill in values.
+
+OAUTH_CLIENT_ID=
+OAUTH_CLIENT_SECRET=
+OAUTH_REDIRECT_URI=
+F3_API_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The F3_API_BASE_URL key should go before the OAUTH_CLIENT_ID key

(UnorderedKey)